client/src/views/landing.vue

```diff
@@ -142,4 +156,7 @@ export default class Landing extends Vue {
 .v-tooltip__content {
   pointer-events: auto !important;
 }
+.tooltip-extra-wide {
```

Comment:
LOL

Reply:
Also check the minimum width for this tooltip (ie, narrow your window and verify that content wraps correctly).

Reply:
For sure, though the view as a whole doesn't quite compress that small.